docs/application/dotnet/get-started/wearable/first-xaml-app.md

```diff
@@ -126,6 +130,30 @@ If you face any issues during deployment, it is recommended to manually install
 ```bash
 $ sdb install <path-to-package>/org.tizen.example.TizenWearableXamlApp1-1.0.0.tpk
 ```
+## Validate Your Application Changes Using XAML Hot Reload
+
+To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, you can now immediately see the UI edits on the running emulator without having to rebuild or redeploy your application. This feature helps you save time and validate your changes easily.
```

Comment:
```suggestion
To test your UI code changes in your application, you must build and deploy your application. With the introduction of the XAML Hot Reload feature, the UI edits on the running emulator are applied immediately on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.
```

However, this suggestion is related with https://github.com/Samsung/tizen-docs/pull/1155#discussion_r448718974, and I'm waiting the answer to the question to this comment.

Reply:
@mijin85cho,
As mentioned in the previous comment, if I change "would have" to to must, then in that case we are first telling the user that we "must do" a certain activity. Then, in the immediate next sentence we are saying that now because of the new feature you need not do that activity. So here the usage of must according to me is contradicting and may not give the right message.
If you still suggest to use must, I will change it to "must".

Please let me know if the following is okay:

To test your UI code changes in your application, you would have to build and deploy your application. With the introduction of the XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.

Please let me know your views.

Reply:
@surbhig12 
Thanks for the explanation. I understood your worrying point.
Considering your concern, how about below style?

To test your UI code changes in your application, you must build and deploy your application. However, if you use XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without rebuild or deploying your application again. It helps you save time and validate the changes easily.

Reply:
@mijin85cho,

Thank you for understanding.
I suggest:

To test your UI code changes in your application, you must build and deploy your application. However, if you use the XAML Hot Reload feature, the UI edits are immediately applied on the running emulator without building or deploying your application again. It helps you save time and validate the changes easily.

Please verify

Reply:
I agree with your rewriting. Thanks for the modification.